### PR TITLE
Initial integration with QC Tasks

### PR DESCRIPTION
**Pull request by knopers8**
This adds 5 QC Tasks, which can be enabled with --include-qc flag passed to o2dpg_sim_workflow.py. In this iteration they publish results directly to QCDB (at ccdb-test) and results from many TFs and sub-jobs are not merged.

More Tasks to come, also merging will be done in future PRs.

An example to run it can be found in `MC/run/examples/O2DPG_pp_minbias_one_tf_qc.sh`

### CODE REVIEW

**Comment by knopers8** [comment]
@sawenzel Does O2DPG require explicit dependencies on O2 and now also QC, or it just requires users to load what is needed manually?

@davidrohr @chiarazampolli  @tomas-herman @tmtetwa FYI, this is some initial effort in integrating QC with the simulation data challenge scripts. I added the MFT Clusters and Tracks QC, TPC Tracking QC, RECO Vertexing QC and TRD Digits QC as an exercise. Please let me know if you have any suggestions.

**Comment by sawenzel** [comment]
@knopers8 : There is some runtime dependence. One should make sure to say
`alienv enter O2,O2DPG,QC`  or put QC into the `O2sim` meta package. So I think nothing special needs to be done.

File: MC/run/examples/O2DPG_pp_minbias_one_tf_qc.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+#
+# A example workflow MC->RECO->AOD for a simple pp min bias 
+# production
+
+# make sure O2DPG + O2 is loaded
+[ ! "${O2DPG_ROOT}" ] && echo "Error: This needs O2DPG loaded" && exit 1
+[ ! "${O2_ROOT}" ] && echo "Error: This needs O2 loaded" && exit 1
```
Comment (sawenzel) [inline MC/run/examples/O2DPG_pp_minbias_one_tf_qc.sh:9]: maybe one could add a check for `QC_ROOT`  or similar here

Comment (knopers8) [reply MC/run/examples/O2DPG_pp_minbias_one_tf_qc.sh:9]: Indeed, I will add this, thanks.

**Pull request merged**